languages/nim/exercises/concept/basics/.meta/example.nim

```diff
@@ -0,0 +1,18 @@
+const expectedMinutesInOven* = 40
+
+const preparationMinutesPerLayer = 2 ## \
```

Comment:
Is the `## \` a typo?

Reply:
No it's not, see [my comment above](https://github.com/exercism/v3/pull/2214#discussion_r477610539). Its for telling the doc generation that the following doc comment is for the preceding `const`